File: swagger-client/scala-akka-http-server/src/main/scala/io/swagger/server/model/EmailAddressParam.scala
package io.swagger.server.model


/**
 * = EmailAddressParam =
 *
 * A single email address parameter used as the body of `POST /mail/blocks/delete`.
 *
 * @param email The email address to delist from all block lists. for example: ''user@domain.com''
 */
case class EmailAddressParam (
  email: String
)

File: openapi-client/javascript-closure-angular/API/Client/EmailAddressParam.js
goog.provide('API.Client.EmailAddressParam');

/**
 * A single email address parameter used as the body of `POST /mail/blocks/delete`.
 * @record
 */
API.Client.EmailAddressParam = function() {}

/**
 * The email address to delist from all block lists.
 * @type {!string}
 * @export
 */
API.Client.EmailAddressParam.prototype.email;

